### DISCUSSION TITLE
Allure Test Results are not appearing after uploading it into Allure Testops

### DISCUSSION THREAD

**Discussion by Badi-Uz-Zaman** [allure-framework/allure2#2320] [Bug Reports] [answered]
I installed the VS Code plugin, ran my tests using Playwright, and generated test results. While successfully uploading these results to Allure TestOps, there's nothing showing up inside the launch section.

**Comment by cheshi-mantu** [comment]
Hi @Badi-Uz-Zaman 
Could you please shed more light here? Screenshots or screen recording could help.

Cheers

**Reply by GlebBakirov1** [reply]
@cheshi-mantu 
Hi Egor! I have the same problem. 
Screencast is attached

https://github.com/allure-framework/allure2/assets/146427914/5927a4a3-4201-43c4-86c0-9047f2d8aaa0

**Reply by cheshi-mantu** [reply]
We've highlighted this issue to the VSCode plug-in developer. In case we'll have questions, we'll get back to you guys.

**Comment by epszaw** [comment, marked as answer]
Hey there! We published a new version of the extension (`1.0.12`) which includes the potential fix. Could you update the extension and try to create a launch again?

**Reply by GlebBakirov1** [reply]
I updated extension and checked - it works! Thank you!

**Reply by epszaw** [reply]
Glad to help you! :)
@Badi-Uz-Zaman hope it'll help you too.
